libs/ts-rest/core/src/lib/dsl.spec.ts

```diff
@@ -477,4 +477,27 @@ describe('contract', () => {
       >
     >;
   });
+
+  describe('pathPrefix', () => {
+    it('Should recursively apply pathPrefix to path', () => {
```

Comment:
Might be useful to add a type assertion test just like the test above to make sure the runtime path and TS type are the same